platform/security/handler/security-handler-oidc/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,29 +14,38 @@
 -->
 <blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.2.0"
-           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
            xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
            xsi:schemaLocation="
   http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
 
-    <reference id="sessionFactory" interface="ddf.security.http.SessionFactory"
-               filter="(id=http)"/>
-
-    <cm:cm-properties id="initialProperties"
-      persistent-id="org.codice.ddf.security.handler.api.OidcHandlerConfiguration"/>
-
     <bean id="oidcConfiguration" class="org.codice.ddf.security.handler.oidc.OidcHandlerConfigurationImpl">
         <cm:managed-properties persistent-id="org.codice.ddf.security.handler.api.OidcHandlerConfiguration"
                                update-strategy="component-managed" update-method="update"/>
```

Comment:
❗️ Should the update method here be changed? I know before blueprint would call the update method with an updated `Map<String,Object>` properties object, that we would copy to our own properties object.